src/app/features/exchange-set/ess-list-encs/ess-list-encs.component.ts

```diff
@@ -127,6 +129,10 @@ export class EssListEncsComponent implements OnInit {
   displaySingleEnc() {
     this.displaySingleEncVal = true;
   }
+  getAverageSizeofENC() {
```

Comment:
Please rename to `getEstimatedTotalSize`
```suggestion
  getEstimatedTotalSize() {
```